packages/app/src/app/pages/Dashboard/Content/routes/Sandboxes/EmptyState.tsx

```diff
@@ -0,0 +1,185 @@
+    track('Sandboxes - open template from empty state', {
+      codesandbox: 'V1',
+      event_source: 'UI',
+    });
+
+    window.open(url, '_blank');
+  };
+
+  const handleSelectTemplate = (template: TemplateFragment) => {
+    const { sandbox } = template;
+    const collection = dashboard.allCollections?.find(c => c.path === '/');
+
+    track('Sandboxes - select template from empty state', {
+      codesandbox: 'V1',
+      event_source: 'UI',
+    });
+
+    actions.editor.forkExternalSandbox({
+      sandboxId: sandbox.id,
+      openInNewWindow: false,
+      body: {
+        collectionId: collection?.id,
+      },
+    });
+
+    actions.modals.newSandboxModal.close();
+  };
+
+  return (
+    <Stack
+      css={{
+        width: `calc(100% - ${2 * GUTTER}px)`,
+        maxWidth: GRID_MAX_WIDTH - 2 * GUTTER,
+        margin: '24px auto 0',
+      }}
+      direction="vertical"
+      gap={10}
+    >
+      {isPersonalSpace ? (
+        <StyledEmptyDescription
+          as="p"
+          dangerouslySetInnerHTML={{ __html: DESCRIPTIONS.PERSONAL }}
+        />
+      ) : (
+        <Stack
+          css={{
+            width: '100%',
+            '@media screen and (min-width: 768px)': {
+              width: '560px',
+            },
+          }}
+          direction="vertical"
+          gap={6}
+        >
+          <StyledEmptyDescription
+            as="p"
+            dangerouslySetInnerHTML={{ __html: DESCRIPTIONS.TEAM }}
+          />
+        </Stack>
+      )}
+      <Stack direction="vertical" gap={6}>
+        <Text as="h2" margin={0} lineHeight="25px" size={16} weight="normal">
+          Start from a template
+        </Text>
+        <Element
+          as="ul"
+          css={{
+            margin: 0,
+            padding: 0,
+            position: 'relative',
+            overflow: 'hidden',
+            display: 'grid',
+            listStyle: 'none',
+            gap: '16px',
+            gridTemplateColumns: 'repeat(auto-fill, minmax(268px, 1fr))',
+            gridAutoRows: 'minmax(156px, 1fr)',
+          }}
+        >
+          {officialTemplates.state === 'loading'
+            ? new Array(4).fill(undefined).map((_, i) => (
+                <SkeletonText
+                  // eslint-disable-next-line react/no-array-index-key
+                  key={`templates-skeleton-${i}`}
+                  css={{
+                    width: '100%',
+                    height: '100%',
+                  }}
+                />
+              ))
+            : null}
+          {filteredTemplates && filteredTemplates.length > 0
+            ? filteredTemplates.map(template => (
+                <Stack as="li" key={template.id}>
+                  <TemplateCard
+                    key={template.id}
+                    disabled={hasMaxPublicSandboxes}
+                    template={template}
+                    onOpenTemplate={handleOpenTemplate}
+                    onSelectTemplate={handleSelectTemplate}
```

Comment:
I was really confused about what `handleSelectTemplate` would do, because it doesn't seem selectable? What I think happens here is that user / team template logic leaks into the official template buttons. These cards don't need all the functionality and, imo should be (relatively) simple links.

What I'd suggest here is that we create a different component for these (official) template links. The template card feels like a bit like Swiss knife to me this way. The modal close in the `handleSelectTemplate` function also doesn't feel like it belongs.

Reply:
@tristandubbeld I guess I'd rather have the template card behave in a predictable way across the app than look the same but perform different actions in different places.

Reply:
But it doesn't behave the same, right? That's what confused me because it's not selectable. I'm confused now what `TemplateCard` should be used for, only for templates to create a new sandbox from or also for template sandboxes that are editable? If it's only the former I don't think it needs an `onSelectTemplate` function because it doesn't seem to do anything.

Reply:
@tristandubbeld I think the naming is off, but selecting a template actually forks it, and opening a template (using cmd + click/cmd + open) just opens it in a new tab.

Reply:
Interesting, do we actually want that to be the behaviour?

Reply:
@tristandubbeld it's been like this forever I think, lol. I've added tracking to see if people ever just open a sandbox.

Reply:
Nice! Let's see 😄